cloud.foundry.cli/src/main/java/cloud/foundry/cli/logic/ApplyLogic.java

```diff
@@ -38,12 +41,14 @@ public ApplyLogic(@Nonnull DefaultCloudFoundryOperations cfOperations) {
         this.cfOperations = cfOperations;
     }
 
-    //TODO update the documentation as soon as the method does more than just creating applications
     /**
-     * Creates all provided applications that are not present in the live system. In case of any error, the procedure
-     * is discontinued.
-     * @param desiredApplications the applications that should all be present in the live system after the procedure
-     * @throws ApplyException if an error occurs during the procedure
+     * Apply the differences between the apps given in the yaml file and the
+     * configuration of the apps of your cf instance. In case of any error, the
+     * procedure is discontinued.
```

Comment:
```suggestion
     * configuration of the apps of your cf instance. In case of any non-recoverable error, the
     * procedure is discontinued.
```